src/logic/operate.js

```diff
@@ -0,0 +1,20 @@
+import Big from 'big.js';
+
+// eslint-disable-next-line consistent-return
+const operate = (numberOne, numberTwo, operation) => {
+  // eslint-disable-next-line default-case
```

Comment:
- [ ] We need to address the Eslint error here. Check this [link](https://eslint.org/docs/rules/default-case)